packages/product-components/src/components/TokenSummaryIcon/TokenSummaryIcon.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import { Meta, StoryObj } from '@storybook/react';
+import {
+    TokenSummaryIcon as TokenSummaryIconComponent,
+    TokenSummaryIconProps,
+} from './TokenSummaryIcon';
+
+const TOKEN_1 = {
+    balance: '20',
+    contract: '0xdAC17F958D2ee523a2206206994597C13D831ec7',
+    decimals: 6,
+    name: 'Tether USD',
+    symbol: 'usdt',
+    type: 'ERC20',
```

Comment:
can you specify just really required info here?
contract, name, symbol is need right?

Reply:
decimals are required by the type too. i removed the non-required fields.